src/style.css

```diff
@@ -1 +1,98 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
```

Comment:
Consider specifying a more descriptive font-family name instead of 'Avenir' to avoid potential conflicts if other stylesheets also define a font with the same name.